src/templates/content.js

```diff
@@ -53,6 +54,16 @@ export const query = graphql`
               }
             }
           }
+          socialCard {
+            description {
+              id
```

Comment:
Should this be `description` instead of `id`?

Reply:
Yes! Great eye, thank you. Fixed in 3d6aa0c